src/components/SearchForm.jsx

```diff
@@ -16,8 +15,7 @@ const SearchForm = ({
           placeholder="seperate search terms with a comma (,)"
         />
       </label>
-      {touched.searchTerms && errors.searchTerms &&
-      <p className="form-error">{errors.searchTerms}</p>}
+      <ErrorMessage name="searchTerms" />
```

Comment:
Formik has this built-in <ErrorMessage> component that only needs the <Field> component's name and then handles the rest of the logic.  Just makes the code a little sleeker.